src/containers/shared/utils.js

```diff
@@ -23,7 +23,7 @@ export const BAD_REQUEST = 400;
 export const DECIMAL_REGEX = /^\d+$/;
 export const RIPPLE_ADDRESS_REGEX = /^r[rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz]{27,35}$/;
 export const HASH_REGEX = /[0-9A-Fa-f]{64}/i;
-export const CURRENCY_REGEX = /^[a-zA-Z]{3,}\.r[rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz]{27,35}$/;
+export const CURRENCY_REGEX = /^[a-zA-Z0-9]{3,}\.r[rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz]{27,35}$/;
```

Comment:
👍  - wondering though whether we can get these regexes from `xrpl`

Reply:
That's a good idea - I honestly just modified these regexes myself